--- nethack-3.4.3/sys/unix/Makefile.doc.destdir~	2011-09-03 01:34:34.405290601 +0200
+++ nethack-3.4.3/sys/unix/Makefile.doc	2011-09-03 01:34:34.420290619 +0200
@@ -41,7 +41,7 @@ Guidebook.dvi:	Guidebook.tex
 
 
 GAME	= nethack
-MANDIR	= /usr/man/man6
+MANDIR	= /usr/share/man/man6
 MANEXT	= 6
 
 # manual installation for most BSD-style systems
@@ -61,7 +61,7 @@ manpages:
 	-$(GAMEMANCREATE) $(MANDIR)/$(GAME).$(MANEXT)
 	-$(LEVMANCREATE) $(MANDIR)/lev_comp.$(MANEXT)
 	-$(DGNMANCREATE) $(MANDIR)/dgn_comp.$(MANEXT)
-	-$(RCVRMANCREATE) $(MANDIR)/recover.$(MANEXT)
+	-$(RCVRMANCREATE) $(MANDIR)/nethack-recover.$(MANEXT)
 	-$(DLBMANCREATE) $(MANDIR)/dlb.$(MANEXT)
 
 # manual creation for distribution
@@ -83,6 +83,8 @@ recover.txt	: recover.6
 dlb.txt		: dlb.6
 	nroff -man dlb.6 | $(COLCMD) > dlb.txt
 
+install:
+	install -c -D -m644 nethack.6 $(DESTDIR)$(MANDIR)/nethack.6
 
 clean:
 	-rm -f Guidebook.aux Guidebook.log
--- nethack-3.4.3/sys/unix/Makefile.top.destdir~	2003-12-08 00:39:13.000000000 +0100
+++ nethack-3.4.3/sys/unix/Makefile.top	2011-09-03 02:39:39.073868620 +0200
@@ -87,7 +87,7 @@ DAT = $(DATNODLB) $(DATDLB)
 $(GAME):
 	( cd src ; $(MAKE) )
 
-all:	$(GAME) recover Guidebook $(VARDAT) dungeon spec_levs check-dlb
+all:	$(GAME) nethack-recover Guidebook $(VARDAT) dungeon spec_levs check-dlb
 	@echo "Done."
 
 # Note: many of the dependencies below are here to allow parallel make
@@ -172,8 +172,8 @@ dlb:
 
 # recover can be used when INSURANCE is defined in include/config.h
 # and the checkpoint option is true
-recover: $(GAME)
-	( cd util ; $(MAKE) recover )
+nethack-recover: $(GAME)
+	( cd util ; $(MAKE) nethack-recover )
 
 dofiles:
 	target=`sed -n					\
@@ -183,38 +183,44 @@ dofiles:
 		-e '}' 					\
 	  	-e '$$s/.*/nodlb/p' < dat/options` ;	\
 	$(MAKE) dofiles-$${target-nodlb}
-	cp src/$(GAME) $(GAMEDIR)
-	cp util/recover $(GAMEDIR)
+	install src/$(GAME) -D $(DESTDIR)$(GAMEDIR)/$(GAME)
+	install util/nethack-recover -D $(DESTDIR)$(GAMEDIR)
 	-rm -f $(SHELLDIR)/$(GAME)
 	sed -e 's;/usr/games/lib/nethackdir;$(GAMEDIR);' \
 		-e 's;HACKDIR/nethack;HACKDIR/$(GAME);' \
 		< sys/unix/nethack.sh \
 		> $(SHELLDIR)/$(GAME)
 # set up their permissions
-	-( cd $(GAMEDIR) ; $(CHOWN) $(GAMEUID) $(GAME) recover ; \
-			$(CHGRP) $(GAMEGRP) $(GAME) recover )
-	chmod $(GAMEPERM) $(GAMEDIR)/$(GAME)
-	chmod $(EXEPERM) $(GAMEDIR)/recover
-	-$(CHOWN) $(GAMEUID) $(SHELLDIR)/$(GAME)
-	$(CHGRP) $(GAMEGRP) $(SHELLDIR)/$(GAME)
-	chmod $(EXEPERM) $(SHELLDIR)/$(GAME)
+	-if [ -z "$(DESTDIR)" ]; then \
+	-( cd $(GAMEDIR) ; $(CHOWN) $(GAMEUID) $(GAME) nethack-recover ; \
+			$(CHGRP) $(GAMEGRP) $(GAME) nethack-recover ) \
+	-$(CHOWN) $(GAMEUID) $(SHELLDIR)/$(GAME) \
+	$(CHGRP) $(GAMEGRP) $(SHELLDIR)/$(GAME) \
+	fi
+	chmod $(EXEPERM) $(DESTDIR)$(SHELLDIR)/$(GAME)
+	chmod $(GAMEPERM) $(DESTDIR)$(GAMEDIR)/$(GAME)
+	chmod $(EXEPERM) $(DESTDIR)$(GAMEDIR)/nethack-recover
 
 dofiles-dlb: check-dlb
 	( cd dat ; cp nhdat $(DATNODLB) $(GAMEDIR) )
 # set up their permissions
+	-if [ -z "$(DESTDIR)" ]; then \
 	-( cd $(GAMEDIR) ; $(CHOWN) $(GAMEUID) nhdat $(DATNODLB) ; \
-			$(CHGRP) $(GAMEGRP) nhdat $(DATNODLB) ; \
-			chmod $(FILEPERM) nhdat $(DATNODLB) )
+			$(CHGRP) $(GAMEGRP) nhdat $(DATNODLB) ) \
+	fi
+	chmod $(FILEPERM) $(DESTDIR)$(GAMEDIR)/nhdat $(DESTDIR)$(GAMEDIR)/$(DATNODLB)
 
 dofiles-nodlb:
 # copy over the game files
 	( cd dat ; cp $(DAT) $(GAMEDIR) )
 # set up their permissions
+	-if [ -z "$(DESTDIR)" ]; then \
 	-( cd $(GAMEDIR) ; $(CHOWN) $(GAMEUID) $(DAT) ; \
-			$(CHGRP) $(GAMEGRP) $(DAT) ; \
-			chmod $(FILEPERM) $(DAT) )
+			$(CHGRP) $(GAMEGRP) $(DAT) ; ) \
+	fi
+	chmod $(FILEPERM) $(DESTDIR)$(DAT) 
 
-update: $(GAME) recover $(VARDAT) dungeon spec_levs
+update: $(GAME) nethack-recover $(VARDAT) dungeon spec_levs
 #	(don't yank the old version out from under people who're playing it)
 	-mv $(GAMEDIR)/$(GAME) $(GAMEDIR)/$(GAME).old
 #	quest.dat is also kept open and has the same problems over NFS
@@ -230,23 +236,26 @@ update: $(GAME) recover $(VARDAT) dungeo
 # and a reminder
 	@echo You may also want to install the man pages via the doc Makefile.
 
-install: $(GAME) recover $(VARDAT) dungeon spec_levs
+install: $(GAME) nethack-recover $(VARDAT) dungeon spec_levs
 # set up the directories
 # not all mkdirs have -p; those that don't will create a -p directory
-	-mkdir -p $(SHELLDIR)
-	-rm -rf $(GAMEDIR) $(VARDIR)
-	-mkdir -p $(GAMEDIR) $(VARDIR) $(VARDIR)/save
-	-rmdir ./-p
-	-$(CHOWN) $(GAMEUID) $(GAMEDIR) $(VARDIR) $(VARDIR)/save
-	$(CHGRP) $(GAMEGRP) $(GAMEDIR) $(VARDIR) $(VARDIR)/save
-	chmod $(DIRPERM) $(GAMEDIR) $(VARDIR) $(VARDIR)/save
+	-mkdir -p $(DESTDIR)$(SHELLDIR)
+	-mkdir -p $(DESTDIR)$(GAMEDIR) $(DESTDIR)$(VARDIR) $(DESTDIR)$(VARDIR)/save
+	-if [ -z "$(DESTDIR)" ]; then \
+		-rmdir ./-p ; \
+		-$(CHOWN) $(GAMEUID) $(GAMEDIR) $(VARDIR) $(VARDIR)/save ; \
+		$(CHGRP) $(GAMEGRP) $(GAMEDIR) $(VARDIR) $(VARDIR)/save ; \
+	fi
+	chmod $(DIRPERM) $(DESTDIR)$(GAMEDIR) $(DESTDIR)$(VARDIR) $(DESTDIR)$(VARDIR)/save
 # set up the game files
 	( $(MAKE) dofiles )
 # set up some additional files
-	touch $(VARDIR)/perm $(VARDIR)/record $(VARDIR)/logfile
+	touch $(DESTDIR)$(VARDIR)/perm $(DESTDIR)$(VARDIR)/record $(DESTDIR)$(VARDIR)/logfile
+	-if [ -z "$(DESTDIR)" ]; then \
 	-( cd $(VARDIR) ; $(CHOWN) $(GAMEUID) perm record logfile ; \
-			$(CHGRP) $(GAMEGRP) perm record logfile ; \
-			chmod $(FILEPERM) perm record logfile )
+			$(CHGRP) $(GAMEGRP) perm record logfile ) \
+	fi 
+	chmod $(FILEPERM) $(DESTDIR)$(VARDIR)/perm $(DESTDIR)$(VARDIR)/record $(DESTDIR)$(VARDIR)/logfile
 # and a reminder
 	@echo You may also want to reinstall the man pages via the doc Makefile.
 
--- nethack-3.4.3/sys/unix/Makefile.utl.destdir~	2003-12-08 00:39:13.000000000 +0100
+++ nethack-3.4.3/sys/unix/Makefile.utl	2011-09-03 01:34:34.420290619 +0200
@@ -273,8 +273,8 @@ lintdgn:
 
 #	dependencies for recover
 #
-recover: $(RECOVOBJS)
-	$(CC) $(LFLAGS) -o recover $(RECOVOBJS) $(LIBS)
+nethack-recover: $(RECOVOBJS)
+	$(CC) $(LFLAGS) -o nethack-recover $(RECOVOBJS) $(LIBS)
 
 recover.o: recover.c $(CONFIG_H) ../include/date.h
 
@@ -395,7 +395,7 @@ spotless: clean
 	-rm -f lev_lex.c lev_yacc.c dgn_lex.c dgn_yacc.c
 	-rm -f ../include/lev_comp.h ../include/dgn_comp.h
 	-rm -f ../include/tile.h
-	-rm -f makedefs lev_comp dgn_comp recover dlb
+	-rm -f makedefs lev_comp dgn_comp nethack-recover dlb
 	-rm -f gif2txt txt2ppm tile2x11 tile2img.ttp xpm2img.ttp tilemap
 
 tileedit: tileedit.cpp $(TEXT_IO)
